openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/SYS2/certificate_estimation/SYS2_PRC_variables.py

```diff
@@ -86,11 +86,102 @@ class SYS2_pool_pump_type(Variable):
     metadata = {
         'variable-type' : 'user-input',
         'label' : 'Pool pump type',
-        'display_question' : 'What type of pool pump have you installed',
+        'display_question' : 'What type of pool pump is installed?',
         'sorting' : 5
     }
 
 
+class SYS2_input_power_ESCS_eligibility(Variable):
+    value_type = bool
+    entity = Building
+    definition_period = ETERNITY
+
+
+class SYS2_input_power_PRCS_eligibility(Variable):
```

Comment:
i dont think we need this variable?

not using it anywhere...

Reply:
I'll take a look - think you're right!

Reply:
Yup, removed and all tests passing